src/Options/index.js

```diff
@@ -40,6 +40,75 @@ type RequestKeywordDenylist = {
   value: any,
 };
 
+export interface GraphQLConfig {
+  // All classes enabled by default
```

Comment:
These comments have a defined syntax. Please see the other comments and make sure to run `npm run definitions` to generate the config files.